setup.cfg

```diff
@@ -1,55 +0,0 @@
-[metadata]
-name = adam_core
-version = 0.1.0
-
-[options]
-packages = find:
-python_requires = >=3.9
-install_requires =
-    astropy
-    astroquery
-    healpy
-    jax
-    jaxlib
-    numpy==1.24.4
-    pyarrow>=13.0.0
-    pandas
-    requests
-    scipy
-    spiceypy
-    quivr>=0.7.1
-    mpc-obscodes
-    naif-de440
-    naif-leapseconds
-    naif-eop-high-prec
-    naif-eop-predict
-    naif-eop-historical
-    naif-earth-itrf93
-
-[options.extras_require]
-tests =
-    pytest
-    pytest-cov
-    pytest-benchmark
-    pre-commit
-
-[options.package_data]
-* = *.csv
```

Comment:
I think we need to add the .parquet and .csv files to the package data for hatch as well? `make_real_orbits()` is currently broken on main when you pip install adam_core from source because it can't find the parquet files.

Reply:
Hmm, never mind. `hatch build` packages the .parquet files in the tarball it seems.